src/pydata_sphinx_theme/assets/styles/components/_search.scss

```diff
@@ -58,8 +58,6 @@
   &:focus,
   &:focus-visible {
     border: none;
-    box-shadow: none;
-    outline: 3px solid var(--pst-color-accent);
```

Comment:
The new `:focus-visible` rule in the base stylesheet now provides this focus ring.